src/main/java/org/openstreetmap/atlas/checks/maproulette/MapRouletteCommand.java

```diff
@@ -32,6 +33,10 @@
     private static final Switch<String> PROJECT_DISPLAY_NAME = new Switch<>("projectDisplayName",
             "Display name of the project under which all of the challenges will be submitted",
             StringConverter.IDENTITY);
+    protected static final Switch<String> OUTPUT_PATH = new Switch<>("outputPath",
+            "Full path to file where challenge ids are stored after creation in MapRoulette.",
```

Comment:
"challenge + project ids"

Reply:
Changed it. Have to push the latest code.